products/apps.py

```diff
@@ -0,0 +1,13 @@
+"""
+App settings
+"""
+from django.apps import AppConfig
+
+
+class ProductsConfig(AppConfig):
+    """
+    Product app settings
+    """
+
+    default_auto_field = "django.db.models.BigAutoField"
```

Comment:
Also, can you remove this, I don't think we need this. Only if its easy.

Reply:
Done